src/app/page.tsx

```diff
@@ -1,88 +1,31 @@
+import Tutorial from '@/components/Tutorial'
+import FAQ from '@/components/FAQ'
 
-export default function Index() {
-  const [events, setEvents] = useState<any[]>([])
-
-  useEffect(() => {
-    const getMyEvents = async (creatorId: UUID) => {
-      fetch(`/api/events?creatorId=${creatorId}`, {
-        method: 'GET',
-        headers: {
-          'Content-Type': 'application/json',
-        },
-      })
-        .then((response) => {
-          if (!response.ok) {
-            return response.json().then((err) => {
-              throw new Error(err.message)
-            })
-          }
-          return response.json()
-        })
-        .then((data) => {
-          console.log(data)
-          setEvents(data)
-        })
-        .catch((error) => {
-          console.error('Error:', error.message)
-        })
-    }
-
-    getMyEvents('9e33186f-95db-4385-a974-ee38c8e07547') // filler UUID for now until local storage is setup
-  }, [])
-
+export default function LandingPage() {
   return (
-    <div className="container mx-auto p-4">
-      {' '}
-      {/*main container with max width auto center and padding 4*/}
-      <div className="flex">
-        {' '}
-        {/*flex container to arrange child elements in a row*/}
-        <div className="w-2/3 p-4">
-          {' '}
-          {/*container for left side 2/3 of the width and has padding 4*/}
-          <div className="mb-4 flex items-center justify-start">
-            {' '}
-            {/*flex container to align heading and button horizontally to have them closer together*/}
-            <h1 className="text-2xl font-black font-extrabold">My Events</h1>{' '}
-            {/*heading for MY Events*/}
-            <Link href="/create-event">
-              <button className="btn btn-primary ml-6">+ New Event</button>{' '}
-            </Link>
-            {/*button to add new event with primary styling and margin left 6*/}
-          </div>
-          <div className="grid grid-cols-3 gap-4">
-            {' '}
-            {/*grid container with 2 columns and gap between the items*/}
-            {events.length > 0 &&
-              events.map((event) => (
-                <EventCard
-                  title={event.title}
-                  starttime={event.starttime}
-                  endtime={event.endtime}
-                  key={event.id}
-                />
-              ))}{' '}
-          </div>{' '}
-          {/*div to close grid container*/}
-        </div>{' '}
-        {/*div to close left side container*/}
-        <div className="w-1/3 bg-white p-4">
-          {/*container for right side 1/3 of the width and has padding 4 background white and is left blank on prupose (will add later)*/}
-        </div>
-      </div>{' '}
-      {/*div to close flex container*/}
-    </div>
+    <>
+      <Header />
+      <div className="mb-0 mt-10 pb-0">
+        <Intro />
+        <Image
+          src="/ViewingPage.png"
+          width="1000"
```

Comment:
Probably not the constraint you want because you are forcing this into a square and the aspect ratio is 1592x956 or 1.66.  I'm guessing auto might fix that.  The image itself is very large and a png makes it a slower download. Might be a good option for webm format instead or jpg with 80% or 95% compression.  This is landing page so you want responsive and fast and not have to wait for images to load.  I know this image will be replaced but want to get the concept right.